packages/core/test/unit/data-types/misc-data-types.test.ts

```diff
@@ -200,6 +203,7 @@ describe('DataTypes.JSON', () => {
         default: `'"string"'`,
         mysql: `CAST('"string"' AS JSON)`,
         mssql: `N'"string"'`,
+        oracle: `'string'`,
```

Comment:
I think this is what I remarked in other comments, this does not feel escaped to me

Reply:
Doesn't require escaping.